client/src/components/calender/Calender.tsx

```diff
@@ -68,13 +80,45 @@ const Calendar: FC<Props> = ({ Event, setEvent }): ReactElement => {
       }));
     }
   };
+  const validMatches = [...events, Event];
+
+  const getStadiumMatches = async (id: number | undefined) => {
+    const { data } = await axios.get(`/api/v1/matches/stadium/${id}`);
+    if (data.status === 401) {
+      setValidateError(data.data);
+    }
+    let convertedMatches;
+    if (Array.isArray(data.data)) {
+      convertedMatches = data?.data?.map((event: prevInterface) => {
+        return {
+          title: event.title,
+          start: event.startDate,
+          end: event.endDate,
+          description: event.description,
+          seats: event.seats,
+        };
+      });
+    }
 
+    setMatches(convertedMatches);
+  };
+
+  const { id } = useParams();
   useEffect(() => {
     if (Event) {
       setEvents([...matches]);
       setEvents((prev: IEvent[]) => [...prev, Event]);
     }
   }, [matches]);
+  useEffect(() => {
+    if (type === 'create') {
```

Comment:
no need for nested if  
 if (type === 'create' && StadiumId > 0) {
      getStadiumMatches(StadiumId);
    } else if (type === 'profile' && id) {
      getStadiumMatches(parseInt(id, 10));
    }